connectivity/manifests/allow-all-except-world.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: cilium.io/v2
+kind: CiliumNetworkPolicy
+metadata:
+  namespace: cilium-test
+  name: allow-all-except-world
+spec:
+  endpointSelector: {}
+  egress:
+  - toEntities:
+    - host
+    - remote-node
+    - cluster
+    - init
+    - health
+    - kube-apiserver
+  - toEndpoints:
+    - {}
```

Comment:
Because it doesn't cover other clusters in the clustermesh. I know, I also had the same thought when I discovered the multicluster Ci was failing reliably. Took me some time to realize the mistake. The key was checking the `cilium policy selectors` output.

Reply:
Oh, wow! I had never considered this. Does that mean we'd need a new entity for `clusters` (with a less ambiguous name)?

Reply:
Oh good point. I considered that yesterday, but more of should we expand the definition of `cluster`. I came to the conclusion that that would be confusing, but a new entity of `clusters` is actually an idea worth considering!